app/assets/stylesheets/sequence_builder.css

```diff
@@ -209,8 +202,40 @@ span.prefix{
   font-weight: bold;
 }
 
-/* SITE-WIDE ISH */
+/* SITE-WIDE BACKGROUND */
 
 body.ng-scope {
```

Comment:
I figured this was wrong. I'm trying to make the white background of the whole site a light grey. Probably needs to be a foundation override, huh? I'll look into doing it the right way :)